.travis.yml

```diff
@@ -26,7 +30,7 @@ addons:
       - llvm-7
 
 cache:
-  ccache: false
+  ccache: true
```

Comment:
No, this is worse than without saving ccache (the savings in compile time aren't enough for the time wasted saving and restoring the cache), which is why it was turned off.